coffee_core/src/coffee.rs

```diff
@@ -223,7 +224,23 @@ impl PluginManager for CoffeeManager {
         for repo in self.repos.values() {
             if let Some(mut plugin) = repo.get_plugin_by_name(plugin) {
                 log::trace!("{:#?}", plugin);
-                let result = plugin.configure(verbose).await;
+
+                let source_folder = plugin.root_path.clone();
+                let destination_folder =
+                    format!("{}/{}/plugins", self.config.root_path, self.config.network);
+                dir::copy(
+                    source_folder,
+                    destination_folder.clone(),
+                    &dir::CopyOptions::new(),
+                )
+                .map_err(|err| error!("{}", err.to_string()))?;
```

Comment:
This is not `async` so we are losing the async by blocking this call. It is tricky, but we must write out a version.

Take as an example this code and implement it with tokio

```rust
/// Copy files from source to destination recursively.
pub fn copy_recursively(source: impl AsRef<Path>, destination: impl AsRef<Path>) -> io::Result<()> {
    fs::create_dir_all(&destination)?;
    for entry in fs::read_dir(source)? {
        let entry = entry?;
        let filetype = entry.file_type()?;
        if filetype.is_dir() {
            copy_recursively(entry.path(), destination.as_ref().join(entry.file_name()))?;
        } else {
            fs::copy(entry.path(), destination.as_ref().join(entry.file_name()))?;
        }
    }
    Ok(())
}
```

We have already some utils in the coffee libs, we may wan put another function in there too